src/index.ts

```diff
@@ -0,0 +1,86 @@
+import { serve } from '@hono/node-server'
+import { Hono } from 'hono'
+import { PrismaClient } from '@prisma/client'
+
```

Comment:
Please create a `routes` folder and create two files there `userRoute` and `index(or routes)`. Then when the app will grow you will import all routes into the `routes` and export a single thing and import into the `app(index.ts)`